src/style.css

```
@@ -1 +1,26 @@
-/* add styles here */
+
+body{
+  margin: 0;
+ 
+}
+
+.stars{
+  display: flex;
+}
+
+.stars__star{
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
```

```suggestion
  background: url(images/star.svg) no-repeat;
```

can be a bit simplified